lib/langchain/llm/groq_open_ai.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Langchain::LLM
+  # LLM interface for Groq OpenAI compatible services
+  #
+  # Usage:
+  #    groq = Langchain::LLM::GroqOpenAi.new(
+  #      api_key: ENV["GROQ_API_KEY"],
+  #      llm_options: {},
+  #      default_options: {}
+  #    )
+  class GroqOpenAi < OpenAI
```

Comment:
I would build a standalone integration to Groq directly instead of subclassing OpenAI. I understand that it works but it only works because Groq is trying to mimic the OpenAI interface.

Reply:
@mattlindsey Not use `ruby-openai` gem and not subclass for Groq.